docs/user_guide.md

```diff
@@ -295,6 +299,54 @@ IP2 assetName2
 
 <img alt="system_setting_asset_Ip_mapping_batch" src="images/user guide/system_setting_asset_Ip_mapping_batch2.png">
 
+#### Asset Management
+
```

Comment:
add a few description and limitations on only assets added through UI can be edited here.